src/main/java/com/anchore/jenkins/plugins/anchore/BuildWorker.java

```diff
@@ -838,6 +838,10 @@ private void initializeJenkinsWorkspace() throws AbortException {
         throw new AbortException("Unable to generate a unique identifier for this build due to invalid configuration");
       }
 
+      // ArtifactArchiver.perform() cannot parse file paths with commas, which this will have in some cases, for
+      // example if this is a matrix job. So replace any commas with underscores to separate the matrix values.
+      buildId = buildId.replaceAll(",", "_");
```

Comment:
My only suggestion here would be to use another local/staging variable for the file path or skip saving it entirely and write it directly to jenkinsOutputDirName. `buildId` is an instance member that should reflect the build identifier for the job. It's not currently used anywhere besides here but it's useful if we decide to implement build specific features that requires an awareness of the build ID

Reply:
Ack, I'll make that change.

Reply:
Done